### ISSUE TITLE
[feature] add another eth fork network

### ISSUE THREAD

**Issue by geeks121** [AleRapchan/flash-swap-arbitrage-bot#2]
Great project,,,
i'm wanted for BSC, Polygon and other network fork from Ethereum,, is that possible to implement? if yes.. how?

**Comment by AleRapchan** [comment]
Yes, it is possible, but the roadmap is to migrate the code to GoLang for now.

**Comment by Carl79c** [comment]
Hello, 
I tried to start your bot but failed? What am I doing wrong? Is there anything I need to configure in the code? 
thank you very much

`PowerShell 7.1.4
Copyright (c) Microsoft Corporation.

https://aka.ms/powershell
Type 'help' to get help.

PS C:\Windows\System32> truffle compile

Compiling your contracts...
===========================
> Compiling .\contracts\Migrations.sol
> Artifacts written to C:\Windows\System32\build\contracts
> Compiled successfully using:
   - solc: 0.5.16+commit.9c3226ce.Emscripten.clang

PS C:\Windows\System32> truffle migrate

Compiling your contracts...
===========================
> Compiling .\contracts\Migrations.sol
> Artifacts written to C:\Windows\System32\build\contracts
> Compiled successfully using:
   - solc: 0.5.16+commit.9c3226ce.Emscripten.clang

> Something went wrong while attempting to connect to the network at http://127.0.0.1:7545. Check your network configuration.

Could not connect to your Ethereum client with the following parameters:
    - host       > 127.0.0.1
    - port       > 7545
    - network_id > 5777
Please check that your Ethereum client:
    - is running
    - is accepting RPC connections (i.e., "--rpc" or "--http" option is used in geth)
    - is accessible over the network
    - is properly configured in your Truffle configuration file (truffle-config.js)

Truffle v5.4.12 (core: 5.4.12)
Node v14.18.0
PS C:\Windows\System32> truffle test
Using network 'test'.


Compiling your contracts...
===========================
> Compiling .\contracts\Migrations.sol
> Artifacts written to C:\Users\Olda\AppData\Local\Temp\test--15096-GYcBH4nWKUyW
> Compiled successfully using:
   - solc: 0.5.16+commit.9c3226ce.Emscripten.clang



  0 passing (1ms)

PS C:\Windows\System32>
`
